From 3a3237a296444ef833df3a428918dc15962a16d9 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sun, 20 Mar 2022 20:23:34 +0100
Subject: [PATCH] Replace usage of GTEST_DISALLOW_ASSIGN_

Fixes the following error:

/src/tests/framework/gmock_fixes.h:104:3: error: ISO C++ forbids declaration of 'GTEST_DISALLOW_ASSIGN_' with no type [-fpermissive]
  104 |   GTEST_DISALLOW_ASSIGN_(ActionResultHolder);
      |   ^~~~~~~~~~~~~~~~~~~~~~

Similar to https://github.com/MirServer/mir/pull/2307
---
 tests/framework/gmock_fixes.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/framework/gmock_fixes.h b/tests/framework/gmock_fixes.h
index b112e8bd..2b4de7e7 100644
--- a/tests/framework/gmock_fixes.h
+++ b/tests/framework/gmock_fixes.h
@@ -101,7 +101,7 @@ class ActionResultHolder<std::unique_ptr<T>>
   std::unique_ptr<T> mutable value_;
 
   // T could be a reference type, so = isn't supported.
-  GTEST_DISALLOW_ASSIGN_(ActionResultHolder);
+  void operator=(ActionResultHolder const&) = delete;
 };
 
 }
-- 
2.35.1

